### DISCUSSION TITLE
Modulo d'evacuazione

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#601] [Richieste di aiuto]
Buongiorno come si fanno ad inserire i moduli per l evacuazione

**Comment by Davide031** [comment]
Siccome non riesco

**Comment by Davide031** [comment]
Quando carico i file in personal data moduli e svuto la chache, clicco su file modello e non mi risulta nulla

**Comment by trinko** [comment]
Hai forse modificato il nome del file? 
Tutti i template devono terminare con **.html.twig**

**Comment by Davide031** [comment]
No è uguale

**Comment by Davide031** [comment]
Ecco il file
!
[17023054619823181542861326894725](https://github.com/iisgiua/giuaschool/assets/137722020/de5ebee4-e677-4e55-9329-df8d7491e60a)

**Comment by Davide031** [comment]
Mi dica lei

**Comment by Davide031** [comment]
Se ora clicco evacuazione dall' account di un docente da errore 500
seguirà il log

**Comment by Davide031** [comment]
Quando clicco evaquazione esce error 500
[2023-12-11_16-05.log](https://github.com/iisgiua/giuaschool/files/13637298/2023-12-11_16-05.log)

**Comment by trinko** [comment]
L'errore è nel nome della cartella **moduli**, deve essere minuscolo: **PERSONAL/data/moduli**

**Comment by Davide031** [comment]
No, lo avevo fatto tutto piccolo ma ora se premo va ma come clicco aggiungi dal menù docenti non va più

**Comment by trinko** [comment]
Dal log che hai allegato sembra che l'errore sia dovuto al fatto che manca il file **src\Entity\Docente.php**

**Comment by Davide031** [comment]
Dove lo trovo

**Comment by trinko** [comment]
> Dove lo trovo

Nel file zip di [installazione della 1.5.6](https://github.com/iisgiua/giuaschool/releases/download/v1.5.6/giuaschool-release-v1.5.6.zip)

Però è strano... Se hai solo eseguito l'aggiornamento alla 1.5.6, questo non cancella file.

**Reply by Davide031** [reply]
Esatto è strano domani le farò sapere
